ccd-definition/CaseEventToFields/HearingNotice-nonprod.json

```diff
@@ -168,13 +168,13 @@
     "PageID": "HearingDetails",
     "PageDisplayOrder": 4,
     "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "N"
+    "ShowSummaryChangeOption": "Y"
   },
   {
     "CaseTypeID": "CIVIL",
     "CaseEventID": "HEARING_SCHEDULED",
     "CaseFieldID": "information",
-    "PageFieldDisplayOrder": 1,
+    "PageFieldDisplayOrder": 10,
```

Comment:
Minor change, but this cann still be 1, right?

Reply:
It works for both 1 or 10